From: Miaohe Lin <linmiaohe@huawei.com>
Subject: mm/memcg: fix wrong function name above obj_cgroup_charge_zswap()
Date: Thu, 3 Aug 2023 20:00:21 +0800

The correct function name is obj_cgroup_may_zswap(). Correct the comment.

Link: https://lkml.kernel.org/r/20230803120021.762279-1-linmiaohe@huawei.com
Signed-off-by: Miaohe Lin <linmiaohe@huawei.com>
Acked-by: Johannes Weiner <hannes@cmpxchg.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memcontrol.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/memcontrol.c~mm-memcg-fix-wrong-function-name-above-obj_cgroup_charge_zswap
+++ a/mm/memcontrol.c
@@ -7786,7 +7786,7 @@ bool obj_cgroup_may_zswap(struct obj_cgr
  * @objcg: the object cgroup
  * @size: size of compressed object
  *
- * This forces the charge after obj_cgroup_may_swap() allowed
+ * This forces the charge after obj_cgroup_may_zswap() allowed
  * compression and storage in zwap for this cgroup to go ahead.
  */
 void obj_cgroup_charge_zswap(struct obj_cgroup *objcg, size_t size)
_
